gourmet/sound_gst.py

```diff
@@ -7,13 +7,14 @@
 from gi.repository import Gst
 
 class Player:
-    def __init__ (self):
-        self.player = Gst.ElementFactory.make('playbin2','player')
+    def __init__(self):
+        Gst.init()
+        self.player = Gst.ElementFactory.make('playbin', 'player')
 
-    def play_file (self,path):
+    def play_file(self,path):
         self.player.set_state(Gst.State.NULL)
-        self.player.set_property('uri','file://'+path)
+        self.player.set_property('uri', 'file://'+path)
         self.player.set_state(Gst.State.PLAYING)
 
-    def stop_play (self,path):
+    def stop_play(self,path):
```

Comment:
```suggestion
    def stop_play(self):
```

Reply:
I removed the `stop_play` methods as they are not used.